### ISSUE TITLE
Consider exporting functions individually.

### ISSUE THREAD

**Issue by wtgtybhertgeghgtwtg** [bhoriuchi/async-iterator-from#1]
So
```
import {fromEvent} from 'async-iterator-from';

const myIterator = fromEvent(myEvent);
```
instead of
```
import AsyncIterator from 'async-iterator-from';

const myIterator = AsyncIterator.fromEvent(myEvent);
```
It's a little less writing and can potentially tree-shake better.  I'd be happy to submit a PR if you'd like.

**Comment by bhoriuchi** [comment]
sure, submit a pr

**Comment by bhoriuchi** [comment]
Sorry this took so long. We just had our 3rd child so I haven't had much time. Thanks for the PR. Published v1.0.0 to NPM.
